### DISCUSSION TITLE
Access 'pressed' state in MotiPressable transition?

### DISCUSSION THREAD

**Discussion by hugo-chq** [nandorojo/dripsy#153] [Q&A]
Hi thanks for making this library, it makes animating really simple

Is it possible to access the "pressed" state in MotiPressable in the transition prop, without wrapping around a view and using a hook?

Currently trying to make a PressableOpacity that animates quickly on press, and slowly on release, similar to iOS native topBar buttons

The preference for not using an inner view is to avoid potential issues with handling nested flex styles

iOS button (fast on press, slow on release):
![ezgif-6-a5dedf0c1506](https://user-images.githubusercontent.com/55731698/142350761-92216d9e-638c-4de5-aa50-f6deca91e388.gif)

**Comment by hugo-chq** [comment]
sorry added this to the wrong repo, it was meant for moti 🤦

**Reply by nandorojo** [reply]
Haha yeah i figured, want to reopen there?
